src/particles/plasma/PlasmaParticleContainerInit.cpp

```diff
@@ -189,8 +189,8 @@ InitParticles (const amrex::IntVect& a_num_particles_per_cell,
                 arrdata[PlasmaIdx::ux       ][pidx] = u[0] * c_light;
                 arrdata[PlasmaIdx::uy       ][pidx] = u[1] * c_light;
                 arrdata[PlasmaIdx::psi      ][pidx] = sqrt(1.+u[0]*u[0]+u[1]*u[1]+u[2]*u[2])-u[2];
-                arrdata[PlasmaIdx::x_prev   ][pidx] = 0.;
-                arrdata[PlasmaIdx::y_prev   ][pidx] = 0.;
+                arrdata[PlasmaIdx::x_prev   ][pidx] = x;
+                arrdata[PlasmaIdx::y_prev   ][pidx] = y;
```

Comment:
Can this change have side effects? E.g. is there a change that we test `x_prev == 0`? I couldn't find this in the code, but maybe @SeverinDiederichs can confirm that this is safe?

Reply:
If anything, it should be better this way. Testing `x_prev == 0` to see if something is initialized would not work because it is possible for x to also be zero.